client/app/vps/translations/Messages_fr_FR.xml

```diff
@@ -498,6 +498,42 @@
 
    <translation id="vps_additional_disk_info_fail" qtlid="281706">Erreur lors du chargement d'informations sur les disques additionnels</translation>
 
+   <translation id="vps_tab_cloud_database">Base de données</translation>
+   <translation id="vps_tab_cloud_database_description">Besoin de bases de données fiables, performantes et 100% managées par OVH ?<br/>Commandez notre service Cloud Databases !<br/>Vous disposerez d'une base de données autonome et compatible avec tous vos produits Cloud OVH.<br/>MariaDB, MongoDB, MySQL, PostgreSQL et Redis disponibles !</translation>
+   <translation id="vps_tab_cloud_database_header_name">Nom de l'instance</translation>
+   <translation id="vps_tab_cloud_database_header_version">Version de la base de donnée</translation>
+   <translation id="vps_tab_cloud_database_header_vps_authorized">VPS autorisé</translation>
+   <translation id="vps_tab_cloud_database_header_status">Statut</translation>
+   <translation id="vps_tab_cloud_database_table_empty">Aucune base de données trouvée.</translation>
+   <translation id="vps_tab_cloud_database_add_authorized_ip">Autoriser le VPS à accéder à cette base</translation>
+   <translation id="vps_tab_cloud_database_remove_authorized_ip">Retirer l'authorisation d'accéder à cette base</translation>
```

Comment:
`authorisation` => `autorisation` in french

Reply:
:+1:
So much :gb: in me.